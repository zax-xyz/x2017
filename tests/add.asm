FUNC LABEL 0
    MOV REG 0 VAL 5
    MOV REG 2 VAL 10
    ADD REG 0 REG 2
    PRINT REG 0
    PRINT REG 2
    ADD REG 2 REG 0
    PRINT REG 0
    PRINT REG 2
    RET
