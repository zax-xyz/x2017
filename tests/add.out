15
10
15
25
